subset/network/network_tests.py

```diff
@@ -0,0 +1,91 @@
+import subprocess
+import time
+import sys
+
+name_of_tests = ['network.min_send', 'network.application.min_send', 'dhcp.long', 'ntp.update']
+
+arguments = sys.argv
+
+test_request = str(arguments[1])
+
+test_id = -1
+
+for x in range(0, len(name_of_tests) - 1):
+    if test_request == name_of_tests[x]:
+        test_id = x
+
+print('test_id=' + str(test_id))
+
+cap_pcap_file = str(arguments[2])
+#capture.pcap
+
+device_address = '127.0.0.1'
+
+if test_id == 0:
+    device_address = str(arguments[3])
+
+report_filename = 'report.txt'
+
+packets_in_report = 10
+
+tcpdump_display_all_packets = 'tcpdump -n src host ' + device_address + ' -r ' + cap_pcap_file
+tcpdump_display_udp_bacnet_packets = 'tcpdump -n udp dst portrange 47808-47809 ' + cap_pcap_file
+tcpdump_display_arp_packets = 'tcpdump -v arp -r ' + cap_pcap_file
+tcpdump_display_ntp_packets = 'tcpdump dst port 123 -r ' + cap_pcap_file
+
+tests = [tcpdump_display_all_packets, tcpdump_display_udp_bacnet_packets, tcpdump_display_arp_packets, tcpdump_display_ntp_packets]
+
+def shell_command_fb(command, wait_time, terminate_flag):
```

Comment:
fb?  Use a descriptive name/word.

Reply:
feedback, I can extend

Reply:
From the code, looks like it's "execute_shell_command"?  Why feedback in this context?  Function names are typically "actions" that are performed, not nouns that describe what the result is. E.g., if you really wanted to use feedback it would be "get_shell_feedback" -- but "feedback" in this context doesn't seem right either, I think you might mean "result"

Reply:
changed to shell_command_with_result